website/docs/rbs-support.md

```diff
@@ -72,19 +72,18 @@ these differences. Things which are possible to express in RBS syntax have no
 analogue in Sorbet and vice versa. Some examples:
 
 - RBS supports duck typing via interfaces (different from Sorbet's
-  [interfaces](abstract.md)), but
-  [Sorbet does not support duck typing](faq.md#can-i-use-sorbet-for-duck-typed-code),
```

Comment:
Please leave the `.md` suffix. As described in [the Docusaurus docs](https://v1.docusaurus.io/docs/en/doc-markdown#linking-other-documents), the point of using the `*.md` is to have the links be valid both in the rendered website and in the GitHub markdown preview.

Reply:
Makes sense, I dropped the commit that was removing them and added them where needed 👍